book/website/ethical-research/activism/activism-case-study-google.md

```diff
@@ -0,0 +1,53 @@
+(er-activism-case-study-google)=
+# Case study: Google Workers & Project Maven
+
+Tech workers have historically had low levels of unionisation for a variety of different reasons. 
```

Comment:
Can we say what the reasons are? I would include that some roles are comparatively highly-paid, tech workers may not stay with the same company for a long time, lots of tech roles are in the US where unionisation is not common outside of a few sectors - what others are there?

Reply:
Have expanded based on those, but more suggestions for reasons are welcome...